AddOns/BasicFinancials/app/src/codeunit/AssistedSetupBF.Codeunit.al

```diff
@@ -0,0 +1,46 @@
+codeunit 20605 "Assisted Setup BF"
+{
+    Access = Internal;
+
+    var
+        AssistedSetupTxt: Label 'Set up the Business Central Basic extension';
```

Comment:
The name of the extension has changed, so we should update it in all the strings. The new name is Basic Experience.

Set up the Basic Experience extension

Reply:
I'm not convinced that the name "Basic Experience" is a good name. As I see it Microsoft has already used that name in a way. In standard table "Experience Tier Setup" there is already a field with the name "Basic". I have therefore choosed to give the extensions field and the Experience the caption = 'Basic Ext'
I dont know what the purpose is with the Microsoft "Basic"-field. 

I'm afraid of misunderstandings between the standard use of the name "Basic Experience" and this extension. Do you se it as a issue?

![image](https://user-images.githubusercontent.com/47662057/85715182-75507b80-b6eb-11ea-916a-1d82cd0e12db.png)
![image](https://user-images.githubusercontent.com/47662057/85715828-222af880-b6ec-11ea-827c-b87fafd95ffd.png)
![image](https://user-images.githubusercontent.com/47662057/85716131-733aec80-b6ec-11ea-9511-970a3c5c752e.png)